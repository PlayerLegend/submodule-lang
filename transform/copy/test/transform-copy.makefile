test/transform-copy:	\
	src/lang/transform/copy/test/test.o \
	src/lang/tree/tree.o \
	src/window/alloc.o \
	src/log/log.o \
	src/lang/tokenizer/tokenizer.o \
	src/convert/source.o \
	src/convert/fd/source.o \
	src/table/table.o \
	src/window/string.o \
	src/lang/transform/transform.o \
	src/lang/transform/copy/copy.o \
	src/table/string.o \
	src/range/streq.o \
	src/range/range_strdup.o

test/transform-copy.run: src/lang/transform/copy/test/run.sh
lang-transform-tests: test/transform-copy
lang-transform-tests: test/transform-copy.run
lang-tests: lang-transform-tests
tests: lang-tests

C_PROGRAMS += test/transform-copy
SH_PROGRAMS += test/transform-copy.run

RUN_TESTS += test/transform-copy.run
