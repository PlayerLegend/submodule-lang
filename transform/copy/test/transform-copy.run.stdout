(
	(
		lit [here]
		lit [is]
		lit [a]
		lit [tree]
		lit [to]
		lit [copy]
	)
	(
		lit [it]
		lit [has]
		lit [some]
		lit [branches]
		(
			lit [and]
			lit [some]
			lit [nesting]
		)
		lit [hopefully]
		lit [it]
		lit [gets]
		(
			lit [all]
		)
		lit [the]
		lit [bugs]
		(
			lit [out]
		)
	)
)
(
	(
		lit [here]
		lit [is]
		lit [a]
		lit [tree]
		lit [to]
		lit [copy]
	)
	(
		lit [it]
		lit [has]
		lit [some]
		lit [branches]
		(
			lit [and]
			lit [some]
			lit [nesting]
		)
		lit [hopefully]
		lit [it]
		lit [gets]
		(
			lit [all]
		)
		lit [the]
		lit [bugs]
		(
			lit [out]
		)
	)
)
